artifacts/rhcos/prerelease/rhcos.go

```diff
@@ -0,0 +1,81 @@
+package prerelease
+
```

Comment:
To have a prerelease package inside rhcos is not consistent to the structure in centos/centosstream.

Reply:
Agreed, I think it would be simpler to stick with the current structure and name it `rhcos` and `rhcosprerelease`.